Automattic/jetpack
# VideoPress: Add "publish first video" popover

Part of #27292

#### Changes proposed in this Pull Request:

* Adds state data to check if an uploaded video is the first uploaded video (count goes from 0 to 1) and if the popover was dismissed
* Adds a `PublishFirstVideoPopover` component that encapsulates `ActionPopover` with the logic needed to know whether the popover needs to be shown or not
* Links to the new post page without any query params for now
* Renames some actions to verbs

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the VideoPress dashboard without any uploaded video
* Upload at least one video
* Check that the popover is shown for the uploaded video or, if more than one video were uploaded at the same time, that the popover is shown for the first video that finishes uploading, regardless of start order
* Check that the popover is shown correctly regardless of screen size and Grid/List choice
* Check that once the popover is dismissed, it is not shown again
* Check that the popover is not shown on videos uploaded after the first one
* Delete all videos
* Upload another video
* Check that the popover is shown again, as there were no videos before
* Check that the button links to the new post page

![2022-12-01_12-08-54](https://user-images.githubusercontent.com/8486249/205095875-565b30c5-3437-4d15-9fb4-a6e5c445cba8.png)
![2022-12-01_12-08-42](https://user-images.githubusercontent.com/8486249/205095886-1739bc53-7ae8-4511-b909-0ed776b3f731.png)

https://user-images.githubusercontent.com/8486249/205756191-5c70d402-9e1e-40f0-ac79-9c2f2a8fe63c.mp4
